converter/src/DelphesEDM4HepConverter.cc

```diff
@@ -10,6 +10,7 @@
 #include "edm4hep/ReconstructedParticleCollection.h"
 #include "edm4hep/TrackCollection.h"
 #include "edm4hep/TrackerHitCollection.h"
+#include "edm4hep/CalorimeterHitCollection.h"
```

Comment:
This is already included above.

Reply:
fixed.

Reply:
Hello @andresailer  @tmadlener anything that prevents this to merged?